src/DaPigGuy/PiggyCustomEnchants/enchants/armor/ForcefieldEnchant.php

```diff
@@ -6,86 +6,62 @@
 
 use DaPigGuy\PiggyCustomEnchants\enchants\CustomEnchant;
 use DaPigGuy\PiggyCustomEnchants\enchants\ToggleableEnchantment;
-use DaPigGuy\PiggyCustomEnchants\PiggyCustomEnchants;
+use DaPigGuy\PiggyCustomEnchants\enchants\traits\TickingTrait;
 use pocketmine\entity\object\ItemEntity;
 use pocketmine\entity\projectile\Projectile;
 use pocketmine\inventory\Inventory;
 use pocketmine\item\Item;
 use pocketmine\level\particle\FlameParticle;
 use pocketmine\math\Vector3;
 use pocketmine\Player;
-use pocketmine\scheduler\ClosureTask;
 use pocketmine\scheduler\TaskHandler;
-use ReflectionException;
 
 /**
  * Class ForcefieldEnchant
  * @package DaPigGuy\PiggyCustomEnchants\enchants\armor
  */
 class ForcefieldEnchant extends ToggleableEnchantment
 {
+    use TickingTrait;
```

Comment:
Why not ``use ToggleTrait`` aswell?

Reply:
The ToggleableEnchantment functions are no longer overwritten here, is it still needed?